MariaDB/server
# MDEV-30502: Change lsb-base to sysvinit-utils

- [x] *The Jira issue number for this PR is: MDEV-30502*


## Description
As Debian 12 deprecates lsb-base change lsb-base to sysvinit-utils. Reasoning to this is that LSB init scripts are changes to that package and 10.10 will be LTS and provided by Debian 12 (Bookworm).

Make also changes to  `debian/autobake-debs.sh` to make sure that there is backward compatibility. Version that are still using lsb-base are Debian **10** and **11** also to Ubuntu **18.04,** **20.04**, **22.04** and **22.10**
## How can this PR be tested?
Run `debian/autobake-debs.sh` with Debian 11 and see if `debian/control` has `lsb-base` and it compiles ok.

## Basing the PR against the correct MariaDB version

## Backward compatibility
This has negative compatibility as after this as without using  `debian/autobake-debs.sh` it does not make packages that works with Sysvinit below Debian 12

## PR quality check
- [x] I checked the [CODING_STANDARDS.md](CODING_STANDARDS.md) file and my PR conforms to this where appropriate.
